apps/taxes/src/modules/avatax/maps/avatax-calculate-taxes-map.ts

```diff
@@ -0,0 +1,99 @@
+import { LineItemModel } from "avatax/lib/models/LineItemModel";
+import { TransactionModel } from "avatax/lib/models/TransactionModel";
+import { TaxBaseFragment } from "../../../../generated/graphql";
+
+import { DocumentType } from "avatax/lib/enums/DocumentType";
+import { ChannelConfig } from "../../channels-configuration/channels-config";
+import { taxLineResolver } from "../../taxes/tax-line-resolver";
+import { CalculateTaxesResponse } from "../../taxes/tax-provider-webhook";
+import { CreateTransactionArgs } from "../avatax-client";
+import { AvataxConfig } from "../avatax-config";
+import { mapChannelAddressToAvataxAddress, mapSaleorAddressToAvataxAddress } from "./address-map";
+import { numbers } from "../../taxes/numbers";
+
+export const SHIPPING_ITEM_CODE = "Shipping";
+
+export function mapLines(taxBase: TaxBaseFragment): LineItemModel[] {
```

Comment:
please dont export functions with such generic names. This app can map lines to many different formats for many specific providers. Yes, I can look up the file name, but it leaves me a poor DX.

Instead, you can introduce `class AvataxCalculationMapper` or something like this. I would never need to think about it because the name will be self explaining, you will also protect future changes from bugs, like accidental import this function to Taxjar

Reply:
I see that below you added wrapping object - I'd prefer a class syntax (or UpperCaseObject) but its fine. However, this function should not be exported in that case

Reply:
For now, I removed the export and added it to the wrapping object. I am not quite happy with this approach so I will refactor this whole bit while working on the next PR